ansible/roles/containerd/vars/main.yml

```diff
@@ -0,0 +1,19 @@
+# containerd
+containerd_packages:
+  - containerd.io
+  - containernetworking-plugins
+
+containerd_dependencies:
+  - yum-utils
+  - device-mapper-persistent-data
+  - lvm2  
+
+docker_old_packages: 
+  - docker
+  - docker-client
+  - docker-client-latest
+  - docker-common
+  - docker-latest
+  - docker-latest-logrotate
+  - docker-logrotate
+  - docker-engine
```

Comment:
Do we need for contained such variable content and to check if they are installed ?

Reply:
The old docker installation removal is suggested in the docker installation, that's why those tasks are there.